units/patch_intent.go

```diff
@@ -493,7 +493,10 @@ func (j *patchIntentProcessor) processTriggerAliases(ctx context.Context, p *pat
 	}
 
 	for _, intent := range triggerIntents {
-		if err := j.env.RemoteQueue().Put(ctx, NewPatchIntentProcessor(mgobson.ObjectIdHex(intent.ID()), intent)); err != nil {
+		job := NewPatchIntentProcessor(mgobson.ObjectIdHex(intent.ID()), intent)
+		job.Run(ctx)
+
+		if err := job.Error(); err != nil {
 			return errors.Wrap(err, "problem enqueueing child patch processing")
```

Comment:
the error is now not about _enqueueing_ the processing, but rather in the actual processing

Reply:
good point, thanks!